### ISSUE TITLE
When all Y values are 0, YAxis draws negative values.

### ISSUE THREAD

**Issue by deehuey** [uber/react-vis#675]
Example:

![image](https://user-images.githubusercontent.com/29404735/33529720-d48a3cd8-d842-11e7-9826-23ed8a4360a8.png)

But as soon as one value is populated (>0), it goes as I'd expect to see it. I tried adding tickValues on the YAxis to `[0, 1, 2, 3, 4, 5]` which worked but only once there's a value. If all 0's I get the above result.

![image](https://user-images.githubusercontent.com/29404735/33529732-f67fcc90-d842-11e7-8c85-b7cc669c7449.png)

How do I get those 0 values to render at the bottom?

**Comment by deehuey** [comment]
![image](https://user-images.githubusercontent.com/29404735/33529836-07723a3c-d844-11e7-8c10-ce1b16dad115.png)

It seems to go from -1 to 0, why not 0 to 1 instead?

**Comment by deehuey** [comment]
![image](https://user-images.githubusercontent.com/29404735/33529849-4f2e3c40-d844-11e7-83b9-7a4ded8bed01.png)

I guess i am just trying to figure out how to get it to always display 0 at bottom. It'll never go negative. Cant' find it in the documentation anywhere.

**Comment by deehuey** [comment]
I saw your response in another issue. I saw no mention of `xDomain` in documentation but I tried it out. I got one step closer:

![image](https://user-images.githubusercontent.com/29404735/33529945-c516e01e-d845-11e7-9686-93257edbedd8.png)

This is for `yDomain={[0, 10]}`, but it still shows my `0` values at the 10 mark. ?

**Comment by mcnuttandrew** [comment]
Hey @deehuey, would you mind reproducing this in a webpack bin? it's a little hard to diagonse from pics

**Comment by sandiiarov** [comment]
[There is example ](https://codesandbox.io/s/v0mmlmv2z5)

**Comment by deehuey** [comment]
I put the yDomain on the Axes, not the XYPlot. Where is the documentation for all this? I could not find it anywhere. I couldn't even find usage of the word yDomain in all documentation. Thanks though, that is the fix I was looking for (15 days ago, went with another charting library =( )

**Comment by cades** [comment]
[XYPlog documentation](https://uber.github.io/react-vis/documentation/api-reference/xy-plot) does not mention `yDomain`.

though a example in [this page](https://uber.github.io/react-vis/documentation/general-principles/interaction) uses `yDomain`, it is too implicit.

**Comment by nickharder88** [comment]
@cades check out this [documentation](https://uber.github.io/react-vis/documentation/general-principles/scales-and-data). It describes the scales in a format [name]Domain, which means you can use the same documentation for xDomain and yDomain.
